src/components/Dashboard/DirectorListSm.vue

```diff
@@ -85,6 +92,14 @@ export default class DirectorListSm extends Mixins(CountriesProvincesMixin, Comm
 // Variables
 $avatar-width: 2.75rem;
 
+// Complete filing required styling
+.complete-filing {
+  padding: 0rem 2rem;
```

Comment:
I don't get it. Why do you need padding for both the div ("class="pt-4 pb-4") and the span in it (this line)?

Reply:
I'm not going to worry about this one.

Reply:
Default span behavior doesn't let me influence the vertical spacing.

Updated display attribute on `.complete-filing` so I can keep the padding to the single style.